easybuild/easyconfigs/s/SPAdes/SPAdes-3.14.0-GCC-8.2.0-2.31.1-Python-2.7.15.eb

```diff
@@ -0,0 +1,57 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# 3.9.0:
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = "CMakeMake"
+
+name = 'SPAdes'
+version = '3.14.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://cab.spbu.ru/software/spades/'
+description = """Genome assembler for single-cell and isolates data sets"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = ['http://cab.spbu.ru/files/release%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['18988dd51762863a16009aebb6e873c1fbca92328b0e6a5af0773e2b1ad7ddb9']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('CMake', '3.13.3'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('libreadline', '8.0'),
+    ('Python', '2.7.15'),
+    ('zlib', '1.2.11'),
+]
+
+start_dir = 'src'
+
+separate_build_dir = True
+
+configopts = ' -DBoost_NO_BOOST_CMAKE=ON'
+
+prebuildopts = """
+    # -lrt needed for glibc < 2.17 (won't hurt on >= 2.17)
+    sed -e 's/libdl.so/libdl.so -lrt/' -i.eb projects/hammer/CMakeFiles/spades-hammer.dir/link.txt
+    sed -e 's/libcityhash.a/libcityhash.a -lrt/' -i.eb projects/ionhammer/CMakeFiles/spades-ionhammer.dir/link.txt
```

Comment:
@JackPerdue Why not use a patch file for this?

Reply:
Because a version-specific patch for a two line fix is a major PITA... this is more flexible for future versions

Plus, the files being changed don't exist until after CMake is run in the build dir (thus prebuildopts and not preconfigopts) and vary from system to system.... I think.

Anyway, these was an easy solution easy to maintain.  iT WAS EASY!!! :)

Reply:
You definitely have a point that this is easier to maintain, but I'd like to avoid relying on `sed` for "runtime patching" like we're doing here...

Which means we need to implement something equivalent in the EasyBuild framework, where you can specify a regex + replacement value + file to patch.

Reply:
@boegel @JackPerdue Any updates here?